<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>676</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="root">
   <layout class="QGridLayout" name="rootLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>5</number>
      </property>
      <widget class="MainWindowTopBar" name="topBar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
      </widget>
      <widget class="QWidget" name="middleSpacer" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QFrame" name="bottomBar">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Embedded Resources</string>
            </property>
            <property name="indent">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,2">
              <item>
               <widget class="QToolButton" name="newERButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>+</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="deleteERButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>25</width>
                  <height>26</height>
                 </size>
                </property>
                <property name="text">
                 <string>-</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="editERButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>Edit</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="showAll">
              <property name="minimumSize">
               <size>
                <width>140</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Show All</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="filter">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Filter</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>17</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="styleSheet">
           <string notr="true">QSplitter {
	background: rgba(0,0,0,0);
}</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Local</string>
              </property>
              <property name="indent">
               <number>8</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ERScrollBox" name="local" native="true"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Global</string>
              </property>
              <property name="indent">
               <number>8</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ERScrollBox" name="global" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuhello_to_the_world">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionImport_Model"/>
    <addaction name="actionImport_Narratives"/>
    <addaction name="actionExport_Narratives"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actionOSG_Debug"/>
    <addaction name="actionCamera_Debug"/>
    <addaction name="actionControl_Debug"/>
    <addaction name="actionReload_Style"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Navigation</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionFreeze_Camera"/>
    <addaction name="actionReset_Camera"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionModel_Outliner"/>
    <addaction name="actionEmbedded_Resources"/>
    <addaction name="actionTime_Slider"/>
    <addaction name="actionFont_Color_Styles"/>
    <addaction name="separator"/>
    <addaction name="actionHistory"/>
   </widget>
   <addaction name="menuhello_to_the_world"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuWindows"/>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionImport_Model">
   <property name="text">
    <string>Import Model...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionError">
   <property name="text">
    <string>Error</string>
   </property>
  </action>
  <action name="actionMessage">
   <property name="text">
    <string>Message</string>
   </property>
  </action>
  <action name="actionProgress">
   <property name="text">
    <string>Progress</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOSG_Debug">
   <property name="text">
    <string>OSG Debug</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionCamera_Debug">
   <property name="text">
    <string>Camera Debug</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionFirst_Person_Navigation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>First Person Mode</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionFirst_Person_Mouse">
   <property name="text">
    <string>First Person Mouse</string>
   </property>
  </action>
  <action name="actionObject_Navigation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Object Mode</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="actionLock">
   <property name="text">
    <string>Lock</string>
   </property>
  </action>
  <action name="actionFreeze_Camera">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Freeze Camera</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionReset_Camera">
   <property name="text">
    <string>Reset Camera</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="actionFlight_Navigation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Flight Mode</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionControl_Debug">
   <property name="text">
    <string>Control Debug</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionExport_Narratives">
   <property name="text">
    <string>Export Narratives...</string>
   </property>
  </action>
  <action name="actionImport_Narratives">
   <property name="text">
    <string>Import Narratives...</string>
   </property>
  </action>
  <action name="actionModel_Outliner">
   <property name="text">
    <string>Models</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionTime_Slider">
   <property name="text">
    <string>Time Slider</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionFont_Color_Styles">
   <property name="text">
    <string>Font and Color Styles</string>
   </property>
  </action>
  <action name="actionHistory">
   <property name="text">
    <string>History</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionEmbedded_Resources">
   <property name="text">
    <string>Embedded Resources</string>
   </property>
  </action>
  <action name="actionReload_Style">
   <property name="text">
    <string>Reload Style</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainWindowTopBar</class>
   <extends>QFrame</extends>
   <header>MainWindowTopBar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ERScrollBox</class>
   <extends>QWidget</extends>
   <header>resources/ERScrollBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
